x-pack/plugins/security_solution/public/flyout/right/header.tsx

```diff
@@ -38,20 +52,22 @@ export const PanelHeader: VFC<PanelHeaderProps> = memo(
       <EuiFlyoutHeader
         hasBorder
         css={css`
-          margin-bottom: -24px;
+          margin-bottom: ${flyoutIsExpandable ? '-24px' : '0px'};
         `}
       >
-        <div
-          // moving the buttons up in the header
-          css={css`
-            margin-top: -24px;
-            margin-left: -8px;
-          `}
-        >
-          <ExpandDetailButton />
-        </div>
+        {flyoutIsExpandable && (
```

Comment:
consider adding some unit tests here

Reply:
the only reason I didn't add some is because we didn't have any for this file and I was being lazy.. But you're right, I will add them on Monday!